Packs/GoogleThreatIntelligence/Integrations/GoogleThreatIntelligenceDTMAlerts/README.md

```diff
@@ -0,0 +1,929 @@
+# Google Threat Intelligence - DTM Alerts
+
+This integration allows the creation of incidents based on DTM Alerts from Google Threat Intelligence.
+
+## Configure Google Threat Intelligence - DTM Alerts in Cortex
+
+| **Parameter** | **Description** | **Required** |
+| --- | --- | --- |
+| API Key | See [Acquiring your API key](#acquiring-your-api-key) | True |
+| Fetch incidents |  | False |
+| Max Fetch | Maximum number of Alerts to fetch each time. Maximum value is 25. | False |
+| First Fetch Time | The date or relative timestamp from which to begin fetching Alerts.<br/><br/>Supported formats: 2 minutes, 2 hours, 2 days, 2 weeks, 2 months, 2 years, yyyy-mm-dd, yyyy-mm-ddTHH:MM:SSZ.<br/><br/>For example: 01 May 2025, 01 May 2025 04:45:33, 2025-05-17T14:05:44Z.<br/> | False |
+| Mirroring Direction | The mirroring direction in which to mirror the details. You can mirror "Outgoing" \(from XSOAR to GTI\) direction for DTM Alerts. | False |
+| Alert Type | Fetch Alerts by the specified alert types. | False |
+| Alert Monitor ID | Fetch Alerts by the specified monitor IDs. | False |
+| Alert Status | Fetch Alerts by the specified status. | False |
+| Alert Severity | Fetch Alerts by the specified severity. | False |
+| Alert Tags | Fetch Alerts by the specified tags. | False |
+| Alert Match Value | Fetch Alerts by specified match value. | False |
+| Alert mscore | Fetch Alerts with mscore greater than or equal to the given value.<br/><br/>Note: The value can range in between from 0 to 100. | False |
+| Alert Search | Search Alerts and triggering documents using a Lucene query with text values joined by AND/OR. | False |
+| Use system proxy settings |  | False |
+| Trust any certificate (not secure) |  | False |
+
+### Acquiring your API key
+
+Your API key can be found in your GoogleThreatIntelligence account user menu, clicking on your avatar:
+
+![How to get api key in GoogleThreatIntelligence](../../doc_files/ddeb298-Screen_Shot_2019-10-17_at_3_17_04_PM.png)
```

Comment:
The image referenced here for acquiring an API key appears to be from a different product, not Google Threat Intelligence. The filename `Screen_Shot_2019-10-17_at_3_17_04_PM.png` also suggests it might be outdated. Please replace this with a relevant image showing how to get an API key from the Google Threat Intelligence platform.